// joditor/dist/Store/WriteEditorStore/actions.js
export var SET_LINE_TEXT = "WriteEditor/SET_LINE_TEXT";
export var SET_LINE_TAG = "WriteEditor/SET_LINE_TAG";
export var ADD_LINE = "WriteEditor/ADD_LINE";
export var REMOVE_LINE = "WriteEditor/REMOVE_LINE";
export var REMOVE_LINE_ONLY = "WriteEditor/REMOVE_LINE_ONLY";
export var DROP_LINE = "WriteEditor/DROP_LINE";
export var FOCUS_LINE = "WriteEditor/FOCUS_LINE";
export var SET_IMG = "WriteEditor/SET_IMG";
export var UNSET_IMG = "WriteEditor/UNSET_IMG";
export var DROP_IMG = "WriteEditor/DROP_IMG";
export var SET_TAG_TO_UL = "WriteEditor/SET_TAG_TO_UL";
export var SET_TAG_TO_H1 = "WriteEditor/SET_TAG_TO_H1";
export var SET_TAG_TO_H2 = "WriteEditor/SET_TAG_TO_H2";
export var SET_TAG_TO_H3 = "WriteEditor/SET_TAG_TO_H3";
export var SET_TAG_TO_A = "WriteEditor/SET_TAG_TO_A";
export var SET_TAG_TO_CODE = "WriteEditor/SET_TAG_TO_CODE";
export var SET_TAG_TO_CALL_OUT = "WriteEditor/SET_TAG_TO_CALL_OUT";
export var FOCUS_NEXT_TAG = "WriteEditor/FOCUS_NEXT_TAG";
export var FOCUS_PREV_TAG = "WriteEditor/FOCUS_PREV_TAG";
export var FOCUS_SET_UP_TAG = "WriteEditor/FOCUS_SET_UP_TAG";
export var SEARCH_TAG = "WriteEditor/SEARCH_TAG";
export var FOCUS_NEXT_LINE = "WriteEditor/FOCUS_NEXT_LINE";
export var FOCUS_PREV_LINE = "WriteEditor/FOCUS_PREV_LINE";
export var UNDO = "WriteEditor/UNDO";
export var REDO = "WriteEditor/REDO";
export var SET_BANNER = "WriteEditor/SET_BANNER";
export var SET_TITLE = "WriteEditor/SET_TITLE";
export var RESET = "WriteEditor/RESET";
export var OPEN_TAG_BOX = "WriteEditor/OPEN_TAG_BOX";
export var CLOSE_TAG_BOX = "WriteEditor/CLOSE_TAG_BOX";
export var PUT_SET_UP = "WriteEditor/PUT_SET_UP";
